middlewares/offer/roleValidation.js

```diff
@@ -0,0 +1,14 @@
+const { createError } = require('~/utils/errorsHelper')
+const { VALIDATION_ERROR } = require('~/consts/errors')
+const roleValidation = (req, res, next) => {
+  const { role } = req.user
+  const { FAQ } = req.body
+
+  if (FAQ && role !== 'tutor') {
```

Comment:
why students cannot add FAQ to their requests?

Reply:
According to figma they don't have this option
<img width="249" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-BackEnd/assets/79856961/824d7b7b-17be-48de-a5b6-bc28e99cef96">